Bottom: c779222a11d543e9e4c89261a781f70dc8ac18c3
Top:    1dc0cdb7da1eb430f5ae3243519da57cf531ad6f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-25 13:22:56 +0100

Expose Gathering variable - yuck


---

diff --git a/src/draw.c b/src/draw.c
index dc012cb..5d2cc39 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -71,7 +71,7 @@ RCSID ("$Id$");
  * some local identifiers
  */
 static BoxType Block = {MAXINT, MAXINT, -MAXINT, -MAXINT};
-static bool Gathering = true;
+bool Gathering = true;
 
 static int doing_pinout = 0;
 static bool doing_assy = false;
